web/src/pages/MainMenu.tsx

```diff
@@ -1,5 +1,3 @@
-import React from 'react';
```

Comment:
Im not sure why this is removed, and what this means?  Does it work still?

Reply:
it threw an error when i did tried to run it. later versions of react dont need the import react statement any more i think.